tests/AppTest.php

```diff
@@ -41,4 +44,12 @@ public function testTemplateClassCustom()
             $app->loadTemplate('html.html')
         );
     }
+
+    public function testInitLayoutWithStringArgument()
+    {
+        $app = $this->getApp();
+        $app->initLayout(Admin::class);
```

Comment:
maybe we require standard seed, ie. `[Admin::class]`, thus wont'merge if so

Reply:
yes, then this PR would consist of dropping string param. Though, to be honest, a string param seems quite sensible. At least I didn't see code doing more that just passing class or object, e.g. passing extra parameters.

Reply:
no, we have strict convenction about seeds - always an array with 1st param class name

Reply:
That makes life easy, just removed in comment that string is a valid parameter

Reply:
Happy now :)